site/scripts/prebuild.ts

```diff
@@ -53,6 +53,7 @@ const hookGroupings = {
     "useClientActions",
     "useChain",
   ],
+  "Solana Actions": ["useSolanaTransaction", "useSolanaSignMessage"],
```

Comment:
This was the biggest aha moment

Reply:
@moldy530 What about the provider? Should I make a context or make it part of the other alchemy provider, and should I add this in as a seperate config field, or part of chain setup, which I have to filter now I think for viem vs not